test-data/unit/check-incremental.test

```diff
@@ -5389,3 +5393,58 @@ E = Enum('E', f())  # type: ignore
 [builtins fixtures/list.pyi]
 [out]
 [out2]
+
+[case testChangedPluginsInvalidateCache]
+# flags: --config-file tmp/mypy.ini
+import a
```

Comment:
Would it make sense to add a fine-grained incremental test case where a plugin changes?

Reply:
As we discussed, this will have no effect.